README.md

```diff
@@ -1,134 +1,77 @@
-# iNaturalist-PythonProjectTemplate
+# iNaturalist GeoModel Annotator Project
 
 Code repository for 2024 Data Science for the Common Good project with iNaturalist. 
 
-Collaborators: Angela Zhu, Paula Navarrete, Sergei Pogorelov
+Collaborators: Angela Zhu, Paula Navarrete, Sergei Pogorelov, Ozzie 
 
 This template draws a lot of inspiration from [Cookiecutter Data Science](https://drivendata.github.io/cookiecutter-data-science/). Please read their awesome explanations!
 
 # Getting Started
-## Installation for development
+## :hatched_chick: Installation for local development
 Make sure you update your local branch to the latest.
 
 ```
 $ conda create -n inatator python=3.9
 $ conda activate inatator
 ```
 
-### React components
-1) Navigate to `src/frontend`, run `npm i` that will install js libraries needed for react.
+### React Components
+1) Navigate to `src/frontend`
+2) Run `npm i` that will install js libraries needed for react.
 
-### Backend and SINR
-- Check if you have `poetry`, this is needed because poetry is a tool that finds all compatible library versions while downloading them, otherwise we would die
-- To install poetry go to https://python-poetry.org/docs/
-- It will need you to install pipx, pip on steroids, follow the official pipx instructions to install
-- You don't have to run the command with --global option, the last command
-- Steps will differ slightly based on your OS
+### Backend and SINR Dependency Installation
+1. Check if you have `poetry`, this is needed because poetry is a tool that finds all compatible library versions while downloading them, otherwise we would die
+  - To install poetry go to https://python-poetry.org/docs/
+  - It will need you to install pipx, pip on steroids, follow the official pipx instructions to install
+  - You don't have to run the command with --global option, the last command
+  - Steps will differ slightly based on your OS
+  - After pipx is installed you can install poetry from their instructions, just one line
+2. Check poetry by just typing `poetry` into terminal, it should recognize and list options etc
 
```

Comment:
I believe we are not going to use poetry. In this case, we need to update README to install the requirements on the conda environment. Please either add the requirements.txt file and add the installation of packages in the README.